### PR TITLE
energySpectrActor: added option to normalize histograms to number of simulated events

### PR DESCRIPTION
**Pull request by andiresch**

### CODE REVIEW

File: source/digits_hits/src/GateEnergySpectrumActorMessenger.cc [hunk 1]
```diff
@@ -202,6 +202,12 @@ void GateEnergySpectrumActorMessenger::BuildCommands(G4String base)
   pEnableEnergyPerUnitMassCMD = new G4UIcmdWithABool(bb, this);
   guidance = G4String("Set energy per nucleus");
   pEnableEnergyPerUnitMassCMD->SetGuidance(guidance);
+  
+  
+  bb = base+"/setRelativeToNbPrimaryEvents";
+  pEnableRelativePrimEventsCMD = new G4UIcmdWithABool(bb, this);
+  guidance = G4String("Set values relative to number of primary events");
```
Comment (djboersma) [inline source/digits_hits/src/GateEnergySpectrumActorMessenger.cc]: I think you can provide a better description. Something like "Normalize all enabled 1D histograms to the number of primary events."

File: source/digits_hits/src/GateEnergySpectrumActor.cc [hunk 2]
```diff
@@ -226,7 +228,14 @@ void GateEnergySpectrumActor::Construct()
 /// Save data
 void GateEnergySpectrumActor::SaveData()
 {
-    
+   if (mEnableRelativePrimEvents){
+    for(std::list<TH1D*>::iterator it=allEnabledTH1DHistograms.begin();it!=allEnabledTH1DHistograms.end();++it)
+      {
+          (*it)->Scale(1./nEvent);
+      }
+       //pEnergySpectrumFluenceTrack->Scale(1./nEvent);
```
Comment (djboersma) [inline source/digits_hits/src/GateEnergySpectrumActor.cc]: Could you remove this dead line of code?

File: source/digits_hits/src/GateEnergySpectrumActorMessenger.cc (hunk 1 above)
Comment (andiresch) [reply source/digits_hits/src/GateEnergySpectrumActorMessenger.cc]: True. Changed both, description and flag.

File: source/digits_hits/src/GateEnergySpectrumActor.cc (hunk 2 above)
Comment (andiresch) [reply source/digits_hits/src/GateEnergySpectrumActor.cc]: done.

**Comment by djboersma** [comment]
OK, thanks for the fixes!

**Pull request merged**